LEGO1/lego/legoomni/src/build/legocarbuild.cpp

```diff
@@ -771,14 +793,214 @@ MxS32 LegoCarBuild::FUN_10024850(MxLong p_x, MxLong p_y)
 	return result;
 }
 
-// STUB: LEGO1 0x10024890
```

Comment:
I had to decompile the whole BETA10 function in order to get LEGO1 right, which had some logic differences with LEGO1. Since I didn't want to throw that away, I added it below with an `#ifdef`. I also considered a bunch of inline `#ifdef`s, but that turned out to be quite unreadable. Please tell me if you want me to remove the BETA10 function below.